src/util/bytefile.hpp

```diff
@@ -122,6 +122,8 @@ namespace otto::util {
           return "Exception thrown"; break;
         case Type::PastEnd:
           return "Past the end of the file"; break;
+	default:
```

Comment:
this one again - its very important that there is no default case, so we get the warning if we add a `Type` but forget to add it here

Reply:
Ditto